tracker/tracker_debugger_api.yaml

```
@@ -1318,3 +1374,912 @@ components:
+          description: 'Custom event value'
+          type: number
+          minimum: 0
+          maximum: 1.7976931348623157e+308
+          example: 7.12
+        cip:
+          description: 'Override IP'
+          type: string
+          format: IPv4|IPv6
+          example: '172.16.254.1'
+        cdt:
+          description: ' Override request time'
+          type: string
+          format: UNIX timestamp|UTC DateTime<YYYY-MM-DD hh:mm:ss>
+          example: '2018-02-24 15:34:48'
+        cid:
+          description: 'Configuration ID'
+          type: string
+          minLength: 16
+          maxLength: 16
+          example: '257cc22df0ec483a'
+        _id:
+          description: 'Visitor ID'
+          type: string
+          example: 'dcce8b9ea5d80b52'
+        uia:
+          description: 'Allows to track user anonymously'
+          type: boolean
+          example: true
+        ua:
+          description: 'Override User-Agent header'
+          type: string
+          example: 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36'
+        lang:
+          description: 'Override Accept-Language header'
+          type: string
+          example: 'en-US,en;q=0.9,pl;q=0.8'
+        uid:
+          description: 'User ID'
+          type: string
+          example: 'username-of-some-user'
+        country:
+          description: 'Override GeoIP country'
+          type: string
+          format: ISO 3166-1
+          example: 'pl'
+        region:
+          description: 'Override GeoIP region'
+          type: string
+          example: '85'
+        city:
+          description: 'Override GeoIP city'
+          type: string
+          example: 'Olsztyn'
+        long:
+          description: 'Override GeoIP longitude'
+          type: string
+          example: '20.4952105'
+        lat:
+          description: 'Override GeoIP latitude'
+          type: string
+          example: '53.7683212'
+        cs:
+          description: 'Page charset'
+          type: string
+          example: 'ISO-8859-1'
+        download:
+          description: 'URL of downloaded file'
+          type: string
+          example: 'http://example.com/files/TermsOfService.pdf'
+        link:
+          description: 'External URL opened by the Visitor'
+          type: string
+          example: 'http://help-example.com/how-to/register.html'
+        action_name:
+          description: 'Name of the action'
+          type: string
+          example: 'Piwik PRO Analytics Suite | On-premise & private cloud'
+        search:
+          description: 'Internal search query'
+          type: string
+          example: 'searching for this'
+        search_cat:
+          description: 'Internal search category'
+          type: string
+          example: 'Result category'
+        search_cats:
+          description: 'Result categories'
+          type: string
+          example: 'res cat 1, res cat 2'
```

I think this params should hold json string array.
```suggestion
          example: '["res cat 1","res cat 2"]'
```